node_js/metrics/crashes_checks/index.js

```diff
@@ -0,0 +1,63 @@
+const ElasticSearchService = require('./services/ElasticSearchService');
+const { DEFAULT_ENVIRONMENTS_INFO } = require('./constants');
+const { green, red } = require('./colors');
+const monitoringTools = require('./utils/monitoring_tools');
+const shell = require('shelljs');
+
+const crashesCheck = async (projectName, environmentInfo, monitoringTool) => {
+  if (monitoringTool === 'elastic-apm-node') {
+    const arrayOfData = await Promise.all(
+      environmentInfo.environments.map(environment =>
+        ElasticSearchService.getProjectEnvironmentErrors(projectName, environment)
+          .then(response => {
+            if (response.data.count === 0) {
+              console.log(
+                red,
+                `El ambiente ${environment} del proyecto no tiene errores`
+              );
+            }
+            return Promise.resolve(response.data.count);
+          })
+          .catch(err => {
+            console.log(red, `Hubo un error y no se pudo conectar a kibana en el ambiente ${environment}`);
+            return Promise.resolve(0);
+          })
+      )
+    );
+    const sumOfCrashes = arrayOfData.reduce((acum, value) => acum + value, 0);
+    return Promise.resolve({
+      metric: environmentInfo.metricName,
+      description: environmentInfo.description,
+      value: sumOfCrashes
+    });
```

Comment:
```suggestion
    return {
      metric: environmentInfo.metricName,
      description: environmentInfo.description,
      value: sumOfCrashes
    };
```
It is an async function, it always returns a Promise